### PR TITLE
feat(vps.upscale): add vps le flavors

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | Fix #MANAGER-11454 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by ghyenne — DISMISSED** [review]

File: packages/manager/modules/vps/src/upscale/upscale.constants.js [hunk 1]
```diff
@@ -12,6 +13,9 @@ export const RANGES = {
   ELITE: 'Elite',
   STARTER: 'Starter',
   VALUE: 'Value',
+  LE_16: 'le-16',
```
Comment (darsene) [inline packages/manager/modules/vps/src/upscale/upscale.constants.js:16]: Are these values used ? If not you can remove them from `RANGES` I guess

Comment (brofik) [reply packages/manager/modules/vps/src/upscale/upscale.constants.js:16]: Yes these values are uses in different locations of the code.

File: packages/manager/modules/vps/src/upscale/upscale.constants.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ export const PRICING_MODES = {
   MONTHLY: 'monthly',
   UPFRONT: 'upfront',
 };
+export const LE_RANGES = ['le-2', 'le-4', 'le-16'];
```
Comment (anooparveti) [inline packages/manager/modules/vps/src/upscale/upscale.constants.js:7]: **nitpick:** Avoid duplication
```suggestion
const LE_2 = 'le-2';
const LE_4 = 'le-4';
const LE_16 = 'le-16';
export const LE_RANGES = [LE_2, 'LE_4, LE_16];
```

Comment (brofik) [reply packages/manager/modules/vps/src/upscale/upscale.constants.js]: Fixed

**Review by anooparveti — DISMISSED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Comment by brofik** [comment]
commit pulled to the release branch in the PR https://github.com/ovh/manager/pull/9574

**Pull request closed without merging**